src/renderer/screens/exchange/Swap2/Form/FormSummary/SectionProvider.js

```diff
@@ -25,8 +25,7 @@ const SectionProvider = () => {
         label={t("swap2.form.details.label.provider")}
         details={t("swap2.form.details.tooltip.provider")}
       />
-      {/* TODO: Remove me as soon as the data is connected */}
-      <SummaryValue handleChange={() => {}}>{getProviderIcon("changelly")}</SummaryValue>
+      <SummaryValue value="changelly">{getProviderIcon("changelly")}</SummaryValue>
```

Comment:
Changelly is used has a mock value here, it will be removed as soon as the component will be connected to our state.